### ISSUE TITLE
[Accessibility]: StackedBarChart foreground and background color contrast ratio is not meeting the minimum ratio 3:1

### ISSUE THREAD

**Issue by rileydanejohnston** [carbon-design-system/carbon-charts#1976]
### Application/Team

billing

### Environment

Browser

### What happened? What did you expect to happen instead?

Original issue: https://github.ibm.com/ibmcloud/billing/issues/3391

Component: Graphical chart Bar lines

Issue description: Foreground and back ground color contrast ratio is not meeting the minimum ration 3:1
Note : Same issue is observed with the below pages as well:
1. https://cloud.ibm.com/billing 
Overview page Screen
2. Predicted check box  checked mark fails the contrast ratio as well in the same pages.
 https://cloud.ibm.com/billing 
https://test.cloud.ibm.com/billing

Recommended fix: Foreground and back ground color contrast ratio meet minimum contrast ratio 3:1

User impact: When the background and foreground color does not have a sufficient contrast ratio, it negatively affects the reading experience of users with low vision.

WCAG Mapping: 1.4.11 Non-Text Contrast

Level of Compliance: AA

Severity: Medium

Test tool: Keyboard Firefox

Screen print:

![Image](https://github.com/user-attachments/assets/6defe973-11f8-4490-a9f6-4a166858e163)

### What WCAG 2.1 checkpoint does the issue violate?

Non-Text Contrast

### Version

"@carbon/charts-react": "^1.22.18",

### Data & options used

```js

```

### Relevant log output

```js

```

### What priority level would this issue be in your opinion?

P2

**Comment by theiliad** [comment]
Hi, the grid was not designed to offer that level of contrast. Graph elements (e.g. the bars) were however designed that way
